src/Microsoft.Health.Dicom.Core/Features/HealthCheck/BackgroundServiceHealthCheckCache.cs

```diff
@@ -13,13 +13,17 @@ namespace Microsoft.Health.Dicom.Core.Features.HealthCheck
 {
     public class BackgroundServiceHealthCheckCache
     {
-        private IMemoryCache _cache;
+        private readonly MemoryCache _cache;
 
         private const string OldestDeleteInstanceCacheKey = "_oldestDeleted";
-
         private const string NumDeleteMaxRetryCacheKey = "_numMaxRetries";
 
-        public BackgroundServiceHealthCheckCache(IMemoryCache memoryCache)
+        public BackgroundServiceHealthCheckCache()
```

Comment:
Why this change to the actual implementation?

Reply:
Before it was unintentionally (I think) using a shared memory cache. Instead, I suspect we'll want our own cache for this scenario